### ISSUE TITLE
Would like to use detection only

### ISSUE THREAD

**Issue by grenoblien** [joe-siyuan-qiao/DetectoRS#45]
I am using DetectoRS/mmdetv2 branch for object detection with my own dataset. 

I have edited config files but I am getting errors from masking stage because I don't have mask labels in my dataset. 

How could I edit my config file so that I do not use masking part? 

I am using detectors_htc_r50_1x_coco.py config file.

**Comment by joe-siyuan-qiao** [comment]
If the dataset does not have mask annotations, it may not be a good idea to use HTC-based DetectoRS. Maybe try Cascade-based which does not needs mask inputs.

**Comment by grenoblien** [comment]
Thank you for your reply. 

I didn't know that HTC was for segmentation. Then what is my best option (or config file) for object detection?

**Comment by joe-siyuan-qiao** [comment]
I guess one choice would be using Cascade-RCNN based DetectoRS as shown in the branch mmdetv2.

**Comment by romanvey** [comment]
Can I train Instance Segmentation without a Semantic Segmentation branch in HTC?

**Comment by joe-siyuan-qiao** [comment]
@romanvey Yes. Please refer to configs/htc, which provides details of how to use HTC without semantic segmentation.
